.travis.yml

```diff
@@ -19,5 +20,11 @@ matrix:
   exclude:
     - rvm: 2.4.9
       gemfile: gemfiles/rails_6.0.gemfile
+    - rvm: 3.0.0
+      gemfile: gemfiles/rails_5.0.gemfile
+    - rvm: 3.0.0
+      gemfile: gemfiles/rails_5.1.gemfile
+    - rvm: 3.0.0
+      gemfile: gemfiles/rails_5.2.gemfile
 
```

Comment:
Excluding these `Rails 5.x and Ruby 3 combinations` from the build target.

Because Rails 5 doesn't support Ruby 3.
See: rails/rails#40938